logic/broadcaster.js

```diff
@@ -103,27 +103,19 @@ class Broadcaster {
 	 */
 	getPeers(params, cb) {
 		params.limit = params.limit || this.config.peerLimit;
```

Comment:
We're already using `self` in this module, so we should be consistent.

Reply:
@jondubois you can move the static function `getPeers` outside of the class and use to `self` in order to maintain consistency, otherwise, we have to refactor the `Broadcaster` constructor to use `this` instead of `self`.

Reply:
@ManuGowda I prefer to keep it like this for this PR to keep the diff clean/simple and to avoid conflicts but we can cleanup later once betanet is stable.